templates/patroni.yml.j2

```diff
@@ -61,6 +61,10 @@ bootstrap:
 bypass_api_service: true
 log:
   dir: /var/log/postgresql
```

Comment:
Should the Patroni logs be moved to a separate dir?

Reply:
It can stay here, IMHO.
Reasoning: patroni launches postgresql => newbies will benefits noticing patroni.log IF postgresql failed to start.